tests/unit/test_xapidb_filter.py

```diff
@@ -1,11 +1,13 @@
 # This uses the deprecated imp module because it has to run with Python2.7 for now:
-import imp  # pylint: disable=deprecated-module
+import imp  # pylint: disable=deprecated-module  # pyright: ignore[reportMissingImports]
```

Comment:
It would be imported, but it would only start to be an issue when we run the test with Python12 or newer, where `imp` has been removed. But this import will be updated before that, I hope.

For now, this unit-test is run using Python2.7, and I just moved it to the function using it,
which is better in this case:

Thanks, applied!